utils/.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit=
+    */lib/python*/site-packages/*
+    */encodings/ascii.py
+    */getpass.py
+    */contextlib.py
```

Comment:
I suspect we are hitting an odd bug from nose/coverage.py that is pulling in contextlib and getpass